ggircs-app/app/components/FuelManagement/CarbonTaxActFuelType.tsx

```diff
@@ -0,0 +1,202 @@
+                      <Card.Title>
+                        {carbonTaxActFuelType.carbonTaxFuelType} (
+                        {carbonTaxActFuelType.ctaRateUnits})
+                      </Card.Title>
+                    </Card.Header>
+                    <Card.Body>
+                      <Button
+                        id="show-hide-toggle"
+                        variant="secondary"
+                        onClick={handleShowCharges}
+                      >
+                        {showPeriodText}
+                      </Button>
+                      <Table responsive striped bordered hover size="small">
+                        <thead>
+                          <tr>
+                            <th>Period Start</th>
+                            <th>Period End</th>
+                            <th>Charge</th>
+                            <th>Comments</th>
+                            <th>Edit</th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {carbonTaxActFuelType.fuelChargesByCarbonTaxActFuelTypeId.edges
+                            .filter((node, idx) => idx >= fromIndex)
+                            .map(({ node }) => (
+                              <FuelChargeRow
+                                key={node.id}
+                                fuelId={carbonTaxActFuelType.rowId}
+                                fuelCharge={node}
+                                validateRatePeriod={validateRatePeriod}
+                              />
+                            ))}
+                          <CreateEditFuelChargeRow
+                            operation={"create"}
+                            fuelId={carbonTaxActFuelType.rowId}
+                            connectionId={
+                              carbonTaxActFuelType
+                                .fuelChargesByCarbonTaxActFuelTypeId.__id
+                            }
+                            validateRatePeriod={validateRatePeriod}
+                          />
+                        </tbody>
+                      </Table>
+                    </Card.Body>
+                  </Card>
+                </Col>
+              </Row>
+            </>
+          )}
+          {!carbonTaxActFuelType && (
+            <Row>
+              <Col>
+                Please select a Carbon Tax Act Fuel on the left to view details
+                and related fuel charges
+              </Col>
+            </Row>
+          )}
+        </Col>
+      </Row>
+      <style jsx>{`
+        :global(.bc-card-header) {
+          color: white;
+          background: #003366;
+        }
+        :global(.edit-button) {
+          color: #003366;
+        }
+        :global(.edit-button):hover {
+          color: #000000;
+          cursor: pointer;
+        }
+        :global(.icon-cell) {
+          vertical-align: middle !important;
+          text-align: center;
+        }
```

Comment:
those global styles should be constrained to this component. Maybe replace this outer fragment by a `div`?